courses/machine_learning/deepdive2/structured/solutions/3c_bqml_dnn_babyweight.ipynb

```diff
@@ -0,0 +1,1152 @@
+{
```

Comment:
<p>Retrieve the eval results from either a local variable or a BQ table to avoid recomputing the eval tables (see comment above).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/670/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>Same</p>